.gitignore

```diff
@@ -20,6 +20,8 @@ Makefile.in
 
 .diags.log.meta
 
+.vscode/
```

Comment:
Is this directory created by Visual Studio?
I'm not sure this should be include in this commit. I prefer this is in another Pull-Request.